### PR TITLE
[DOCS-317] update docs with new logger method that accepts a Throwable

### PR DESCRIPTION
**Pull request by jimmyjames**
@vl4ds please review. Once it has a thumbs-up, I will merge this and publish the updated docs when this hits prod.

### CODE REVIEW

File: tools-and-ide/logging.rst [hunk 1]
```diff
@@ -25,16 +25,45 @@ The log instance currently supports these log levels, in decreasing order of sev
 ===== ===================== ====================================================================
 Level Usage                 Description
 ===== ===================== ====================================================================
-ERROR ``log.error(string)`` | Runtime errors or unexpected conditions.
-WARN  ``log.warn(string)``  | Runtime situations that are unexpected, but not wrong.
-                            | Can also be used to log use of deprecated APIs.
-INFO  ``log.info(string)``  | Interesting runtime events. For example, turning a switch on or off.
-DEBUG ``log.debug(string)`` | Detailed information about the flow of the SmartApp.
-TRACE ``log.trace(string)`` | Most detailed information.
+ERROR ``log.error(string)`` Runtime errors or unexpected conditions.
```
Comment (vl4ds) [inline tools-and-ide/logging.rst]: The signatures of these methods are now: log.<level>(String, Throwable = null)

**Comment by vl4ds** [comment]
Also thanks for going out of your way to do this - I completely neglected the documentation when making that change. Will keep this in mind next time I update the API.

**Comment by jimmyjames** [comment]
Pushed another commit to fix the signatures.

**Comment by vl4ds** [comment]
:+1: Looks good

**Pull request merged**